### DISCUSSION TITLE
Compare two folders recursively. Just the files in sub folders ignoring the folder structure.

### DISCUSSION THREAD

**Discussion by prantikv** [WinMerge/winmerge#2167] [Q&A]
I am going to reorganise a lot of files and I do not want any files to go missing. I am using Windows 10. Is there a way I can compare two folders recursively?
As you can see in this video BeyonCompare can do this
https://www.youtube.com/watch?v=iSvdQL8x5Zc

I want to achieve the same thing with WinMerge.
For example, folder 1

MyFiles/
    ├─ folder1
    │  ├─ file1
    │  ├─ file2
    │  ├─ file3
    │  ├─ file4

and it has hundreds of files.

The files will be reorganized based on date so the new folder structure looks like this,
```

MyFiles/
    ├─ folder2
    │  ├─ 2022
    |  │  ├─ file1
    |  │  ├─ file2
    │  ├─ 2023
    |  │  ├─ file3
    |  │  ├─ file4
```
I did try checking the include sub-folder option. That did not work.

I want to list the missing files only from both folders. Or a list that has all the files compared.

I did try winMerge but was not able to figure out how to achieve this as the files are in subfolders.

Do let me know how can this be done on WinMerge. Also do suggest if there is any other software to do this.

Thanks.

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not allow comparisons that ignore folder structure.
However, in [WinMerge2011](https://github.com/datadiode/winmerge2011/releases/tag/0.2011.211.170), which is a fork of WinMerge, it seems that comparisons can be made that ignore the folder structure by clicking "Include Subfolders but ignore folder structure" that appears when you click Include Subfolders several times as shown below.

![image](https://github.com/WinMerge/winmerge/assets/98126/a42cd3dc-930f-4bd1-ae2f-b299630460dc)

**Reply by SummerBeachhh** [reply]
nice,thanks
